distributions/distribution-resources/src/main/resources/userdata/etc/java.util.logging.properties

```diff
@@ -0,0 +1,25 @@
+################################################################################
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I guess not. It was copy pasted. There are a few other cfg files that have it too.